symfony/webpack-encore-bundle/1.0/assets/js/app.js

```diff
@@ -9,6 +9,6 @@
 require('../css/app.css');
 
 // Need jQuery? Install it with "yarn add jquery", then uncomment to require it.
-// var $ = require('jquery');
+// let $ = require('jquery');
```

Comment:
Hm, why not `const $ = require('jquery');`? I think it would be even better because we do not want to override `$`

Reply:
Agree with that. No need to override $ at any moment. I wait @weaverryan review that and I can commit.

Reply:
Yea, const is better - good idea - can you make that update?

Reply:
Done !